packages/admin/admin/src/mui/menu/Item.styles.ts

```diff
@@ -3,7 +3,7 @@ import { createStyles } from "@mui/styles";
 
 import { MenuItemProps, MuiListItemProps } from "./Item";
 
-export type MenuItemClassKey = "root" | "level1" | "level2" | "level3" | "hasIcon" | "hasSecondaryText" | "hasSecondaryAction";
+export type MenuItemClassKey = "root" | "level1" | "level2" | "level3" | "hasIcon" | "hasSecondaryText" | "hasSecondaryAction" | "level3Enumeration";
```

Comment:
I'm not sure the name really explains what this is key is used for... would something like `level3MenuOpen` be more fitting?

Reply:
✅